tests/garage/tf/policies/test_deterministic_mlp_policy_with_model.py

```diff
@@ -0,0 +1,66 @@
+import pickle
+from unittest import mock
+
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.models import Model
+from garage.tf.policies import DeterministicMLPPolicyWithModel
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class SimpleMLPModel(Model):
+    """Simple MLPModel for testing."""
+
+    def __init__(self, name, output_dim, *args, **kwargs):
+        super().__init__(name)
+        self.output_dim = output_dim
+
+    def _build(self, obs_input):
+        return tf.constant(
+            1., shape=(
+                1,
+                self.output_dim,
+            ))
+
+
+class TestDeterministicMLPPolicyWithModel(TfGraphTestCase):
+    def setUp(self):
+        super().setUp()
+        self.env = TfEnv(DummyBoxEnv())
```

Comment:
please test both single- and multi-dimensional obs/action spaces (Dummy defaults to dim=1)